### PR TITLE
Support Generating Mind Map in Word via Addin

### PR DESCRIPTION
**Pull request by Soulike**

### CODE REVIEW

**Comment by Soulike** [comment]
@microsoft-github-policy-service agree company="Microsoft"

File: office-addin/src/taskpane/helpers/WordHelper.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+export class WordHelper {
+  static async getDocumentTitle(): Promise<string> {
+    const defaultTitle = `word_${Math.round((Math.random() + 1) * Date.now()).toString(36)}`;
+
+    return Word.run(async (context) => {
+      const properties = context.document.properties;
+      properties.load("title");
+      await context.sync();
+      return properties.title || defaultTitle;
```
Comment (ZeyuKeithFu) [inline office-addin/src/taskpane/helpers/WordHelper.ts:18]: TODO @ZeyuKeithFu : Fallback to fileUrl

File: office-addin/src/taskpane/helpers/MindMapGenHelper.ts [hunk 2]
```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+ * See LICENSE in the project root for license information.
+ */
+
+/* global URL, Headers, fetch */
+export type MindMapUUID = string;
+export enum DocumentType {
+  DOCX = "docx",
+  PPT = "ppt",
```
Comment (ZeyuKeithFu) [inline office-addin/src/taskpane/helpers/MindMapGenHelper.ts:10]: `pptx`?

**Review by ZeyuKeithFu — APPROVED** [review]

File: office-addin/src/taskpane/helpers/MindMapGenHelper.ts (hunk 2 above)
Comment (Soulike) [reply office-addin/src/taskpane/helpers/MindMapGenHelper.ts:10]: Will fix this in next PR

**Pull request merged**